docs/content/token/staking2/stakers.mdx

```diff
@@ -157,5 +157,5 @@ Transaction arguments:
 |------------|--------|---------|
 | **amount** | UFix64 | The quantity of rewarded FLOW tokens to withdraw from the proxy. |
 
-Since rewards are considered to be unlocked. This marks an equal amount in your locked account as unlocked.
-You can immediately withdraw your unlocked reward tokens from your account after withdrawing rewards.
+Since rewards are considered to be unstaked. This marks an equal amount in your locked account as unstaked.
```

Comment:
```suggestion
Since rewards are considered to be unlocked for the purposes of vesting. This marks an equal amount in your locked account as unlocked.
```